tools/lint.js

```diff
@@ -63,6 +51,50 @@ async function dlint() {
   }
 }
 
+async function dlintPreferPrimordials() {
+  const execPath = getPrebuiltToolPath("dlint");
+  console.log("prefer-primordials");
+
+  const sourceFiles = await getSources(ROOT_PATH, [
+    "runtime/**/*.js",
+    "ext/**/*.js",
+    "core/**/*.js",
+    ":!:core/examples/**",
+  ]);
+
+  if (!sourceFiles.length) {
+    return;
+  }
+
+  const chunks = splitToChunks(sourceFiles, `${execPath} run`.length);
+  for (const chunk of chunks) {
+    const p = Deno.run({
+      cmd: [execPath, "run", "--rule", "prefer-primordials", ...chunk],
+    });
```

Comment:
Why configure here instead of adding to `dlint.json`?

Reply:
If we added `prefer-primordials` to `.dlint.json`, then this rule would apply to almost all JS and TS files in this repository, which would produce lots of diagnostics. This rule should apply only to files under `runtime`, `ext` and `core`, which is why I made it inlined here.